src/lib/openapi/spec/variant-schema.ts

```diff
@@ -5,23 +5,39 @@ export const variantSchema = {
     $id: '#/components/schemas/variantSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'A variant allows for further separation of users into segments. See [our excellent documentation](https://docs.getunleash.io/reference/feature-toggle-variants#what-are-variants) for a more detailed description',
```

Comment:
I feel like saying "our **excellent** documentation" is a little too ... grand (being a Norwegian and all).
But also, the link should explain what bit of the docs its pointing to. And we should also try to avoid using the term "see" to be more inclusive towards vision-impaired people 💁🏼 How about this?

```suggestion
        'A variant allows for further separation of users into segments. Check out the [variants documentation](https://docs.getunleash.io/reference/feature-toggle-variants#what-are-variants) for a more detailed description',
```